src/plugins/Transloadit/index.js

```diff
@@ -457,6 +451,15 @@ module.exports = class Transloadit extends Plugin {
         onAssemblyError(assembly, error)
       }
 
+      const checkAllComplete = () => {
```

Comment:
Moved this check from `onAssemblyFinished` into here, so we don't reject the entire upload anymore if an assembly fails. Instead the files belonging to that assembly are added to the `{ failed }` result array.